src/js/components/CheckBox.js

```diff
@@ -21,30 +42,37 @@ export default class CheckBox extends Component {
         );
       }
     }
+
     if (this.props.className) {
       classes.push(this.props.className);
     }
 
-    return (
-      <label className={classes.join(' ')}
-        aria-describedby={this.props.ariaDescribedby}
-        aria-lebelledby={labelId}>
+    let children = [(
+      <span>
         <input tabIndex="0" className={CLASS_ROOT + "__input"}
-          id={this.props.id} name={this.props.name} type="checkbox"
-          disabled={this.props.disabled}
-          checked={this.props.checked}
-          defaultChecked={this.props.defaultChecked}
-          onChange={this.props.onChange} />
+               id={this.props.id} name={this.props.name} type="checkbox"
+               disabled={this.props.disabled}
+               checked={this.props.checked}
+               defaultChecked={this.props.defaultChecked}
+               onChange={this.props.onChange}
+               ref="input" />
         <span className={CLASS_ROOT + "__control"}>
           <svg className={CLASS_ROOT + "__control-check"} viewBox="0 0 24 24"
-            preserveAspectRatio="xMidYMid meet">
+               preserveAspectRatio="xMidYMid meet">
```

Comment:
I think wrapped lines in tags aligned against the first attribute position.

WebStorm handles those kind of things automatically.